CMake_nRF5x.cmake

```diff
@@ -59,116 +54,210 @@ macro(nRF5x_setup)
             "${NRF5_SDK_PATH}/components/libraries/util/nrf_assert.c"
-            "${NRF5_SDK_PATH}/components/libraries/util/sdk_errors.c"
             "${NRF5_SDK_PATH}/components/libraries/util/app_error.c"
             "${NRF5_SDK_PATH}/components/libraries/util/app_error_weak.c"
+            "${NRF5_SDK_PATH}/components/libraries/util/app_error_handler_gcc.c"
             "${NRF5_SDK_PATH}/components/libraries/util/app_util_platform.c"
-            "${NRF5_SDK_PATH}/components/libraries/log/src/nrf_log_backend_serial.c"
-            "${NRF5_SDK_PATH}/components/libraries/log/src/nrf_log_frontend.c"
             "${NRF5_SDK_PATH}/components/libraries/util/app_util_platform.c"
             "${NRF5_SDK_PATH}/components/libraries/util/sdk_mapped_flags.c"
+            "${NRF5_SDK_PATH}/components/libraries/log/src/nrf_log_backend_serial.c"
+            "${NRF5_SDK_PATH}/components/libraries/log/src/nrf_log_backend_rtt.c"
+            "${NRF5_SDK_PATH}/components/libraries/log/src/nrf_log_default_backends.c"
+            "${NRF5_SDK_PATH}/components/libraries/log/src/nrf_log_frontend.c"
+            "${NRF5_SDK_PATH}/components/libraries/log/src/nrf_log_str_formatter.c"
+            "${NRF5_SDK_PATH}/components/libraries/memobj/nrf_memobj.c"
+            "${NRF5_SDK_PATH}/components/libraries/pwr_mgmt/nrf_pwr_mgmt.c"
+            "${NRF5_SDK_PATH}/components/libraries/ringbuf/nrf_ringbuf.c"
+            "${NRF5_SDK_PATH}/components/libraries/strerror/nrf_strerror.c"
+            "${NRF5_SDK_PATH}/components/libraries/uart/retarget.c"
             )
 
     # Segger RTT
+    include_directories(
+            "${NRF5_SDK_PATH}/external/segger_rtt/"
+    )
+
     list(APPEND SDK_SOURCE_FILES
-            "${NRF5_SDK_PATH}/external/segger_rtt/RTT_Syscalls_GCC.c"
+            "${NRF5_SDK_PATH}/external/segger_rtt/SEGGER_RTT_Syscalls_GCC.c"
             "${NRF5_SDK_PATH}/external/segger_rtt/SEGGER_RTT.c"
             "${NRF5_SDK_PATH}/external/segger_rtt/SEGGER_RTT_printf.c"
             )
 
+
+    # Other external
+    include_directories(
+            "${NRF5_SDK_PATH}/external/fprintf/"
+            "${NRF5_SDK_PATH}/external/utf_converter/"
+    )
+
+    list(APPEND SDK_SOURCE_FILES
+            "${NRF5_SDK_PATH}/external/utf_converter/utf.c"
+            "${NRF5_SDK_PATH}/external/fprintf/nrf_fprintf.c"
+            "${NRF5_SDK_PATH}/external/fprintf/nrf_fprintf_format.c"
+            )
+
+
     # Common Bluetooth Low Energy files
     include_directories(
             "${NRF5_SDK_PATH}/components/ble"
             "${NRF5_SDK_PATH}/components/ble/common"
+            "${NRF5_SDK_PATH}/components/ble/ble_advertising"
+            "${NRF5_SDK_PATH}/components/ble/ble_dtm"
+            "${NRF5_SDK_PATH}/components/ble/ble_link_ctx_manager"
+            "${NRF5_SDK_PATH}/components/ble/ble_racp"
+            "${NRF5_SDK_PATH}/components/ble/nrf_ble_qwr"
+            "${NRF5_SDK_PATH}/components/ble/ble/peer_manager"
```

Comment:
```suggestion
            "${NRF5_SDK_PATH}/components/ble/peer_manager"
```

Reply:
This path was just a mistake when I was updating paths for 15.3.0.